goreleaser/docker.yml

```diff
@@ -110,6 +110,7 @@ dockers:
       - pachyderm/pachd
       - pachyderm/pachd:local
       - "pachyderm/pachd:{{.Version}}"
+      - pachyderm/pachd:{{ .FullCommit }}
```

Comment:
Nice find! although we may want to use `.Env.VERSION` for consistency

Reply:
Chatted about this offline, `FullCommit` makes sense, but we should probably add a note to explain why this is different than `.Version` / `.Env.VERSION`